filters/include/pcl/filters/impl/voxel_grid_covariance.hpp

```diff
@@ -456,6 +456,8 @@ pcl::VoxelGridCovariance<PointT>::getDisplayCloud (pcl::PointCloud<PointXYZ>& ce
   Eigen::Vector3d rand_point;
   Eigen::Vector3d dist_point;
 
+  cell_cloud.reserve (pnt_per_cell * std::count_if (leaves_.begin (), leaves_.end (), [this] (auto& l) { return (l.second.nr_points >= min_points_per_voxel_); }));
```

Comment:
Should we break this in 2 lines?

Reply:
Thanks a lot for your comment.
I'm not so sure about PCL coding rule but is following fine?

```
  cell_cloud.reserve (pnt_per_cell * std::count_if (leaves_.begin (), leaves_.end (),
      [this] (auto& l) { return (l.second.nr_points >= min_points_per_voxel_); }));
```

Reply:
Looks good to me!